roles/kubernetes-apps/ansible/templates/kubedns-deploy.yml

```diff
@@ -83,6 +83,9 @@ spec:
 {% if kube_log_level == '4' %}
         - --log-queries
 {% endif %}
+{% if dns_mode == 'kubedns' %}
```

Comment:
Yeah, I suppose that's true, the request with a bogus domain shouldn't ever make it this far in the dnsmasq mode anyways. Okay, I'll drop that conditional.